README.md

```diff
@@ -65,16 +66,21 @@ bynder_client = BynderClient(
     client_secret='',
     token_saver=token_saver
```

Comment:
completely unrelated to this PR, but I never noticed this was called `token_saver`, why not just `token`?